ckanext/switzerland/templates/user/organizations.html

```diff
@@ -0,0 +1,24 @@
+{% extends "user/read.html" %}
+
+{% block subtitle %}{{ _('Organizations you belong to') }} - {{ super() }}{% endblock %}
```

Comment:
Can you add a German translation (at least) for this please?

Reply:
I changed it to `{{_('Organizations')}}` and chickened out 😇 Hope that is okay, too, as I think it would actually make sense as you can look at that page as yourself but can look at other user's organization page. So Organizations would always be correct ¯\_(ツ)_/¯ 😅